GLTFSDK/Inc/GLTFSDK/ExtensionsKHR.h

```diff
@@ -79,11 +79,13 @@ namespace Microsoft
                 struct TextureTransform : Extension, glTFProperty
                 {
                     TextureTransform();
+                    TextureTransform(const TextureTransform&);
```

Comment:
Why do you need this?

Reply:
Because unique_ptr isn't automatically copyable

Reply:
Unique_ptr only has move semantics so the default copy constructor doesn't work.